<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal_MainWindow</class>
 <widget class="QMainWindow" name="Principal_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>548</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imagenes/logos-UNAH-600x600.png</normaloff>Imagenes/logos-UNAH-600x600.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Agregar_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(146, 196, 125);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:35px;
color:black;</string>
    </property>
    <property name="text">
     <string>Péliculas en total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Acerca_de_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(141, 124, 194);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Acerca de</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Numero_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 70pt &quot;Hack&quot;;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Ver_y_editar_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 217, 102);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Ver y Editar Péliculas</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Visualizacion_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(109, 158, 235);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Visualización de Árbol</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Imagenes/Interfaz.qrc"/>
 </resources>
 <connections/>
</ui>
